src/util/water.ts

```diff
@@ -0,0 +1,418 @@
+import {Path, Circle, Canvas, util} from 'fabric'
+
```

Comment:
Add class documentation. What parameters, such as options and rotor are required. TSDoc looks like a good standard to adopt https://blog.bitsrc.io/documenting-your-typescript-projects-there-are-options-da7c8c4ec554#:~:text=And%20so%2C%20TSDoc%20was%20born.